tests/ftests/083-pybindings-helpers_cgroup_mode.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# Test to excerise cgroup_setup_mode() helpers
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Author: Kamalesh Babulal <kamalesh.babulal@oracle.com>
+#
+
+from cgroup import Cgroup as CgroupCli
+from libcgroup import Cgroup
+import consts
+import ftests
+import sys
+import os
+
+
+def prereqs(config):
+    pass
+
+
+def setup(config):
+    pass
+
+
+def test(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    mode1 = Cgroup.cgroup_mode()
+    mode2 = CgroupCli.get_cgroup_mode(config)
+
+    if mode1 != mode2:
+        result = consts.TEST_FAILED
+        cause = 'mode mismatch: libcgroup mode: {}, tests mode: {}'.format(mode1, mode2)
+        return result, cause
+
+    if mode1 == 1:
```

Comment:
The modes are available in Python via `Mode.CGROUP_MODE_LEGACY`, etc.; we should use them rather than magic numbers.  (You'll need to update the import to be `from libcgroup import Cgroup, Mode`)